src/lib/components/Contributors.svelte

```diff
@@ -10,7 +10,7 @@
   <div class="grid grid-cols-1 gap-6 md:grid-cols-3">
     {#each contributors as contributor, i}
       <div
-        in:fly={{ y: 300, duration: 500, delay: i * 100 }}
+        in:fly|global={{ y: 300, duration: 500, delay: i * 100 }}
```

Comment:
Animation looks nice, but the default navigation to this page puts me at the top of the page so I don't see the animation when I visit the page. Perhaps, it could be made to show once the contributor badges would be visible?

(ig this is just something we can think about later, not really important to do this before merging this pr)

Reply:
That is a valid point, I will also go through and improve typography a bit, so even on this page more stuff will be above the fold on desktop. I will create an issue for this!